examples/neural_networks/plot_elm_diagram.py

```diff
@@ -0,0 +1,86 @@
+"""
+================
+Plot ELM diagram
```

Comment:
This is really not related to ELM that much, right? Maybe by the fact that the output layer has no bias?